assets/script.js

```diff
@@ -72,6 +72,28 @@ function getTrackInfo(userInput) {
       //show the result page after finish call
       $("#track-results-page").show(400);
     }
+    //"reading" the value of <li>
+    // $('#tracks').each(function() {
+      // $('#track-search-result>ol>li').on('click', function() {
```

Comment:
Are you going to use this, if not remove old comments and clean up code.

Reply:
done in latest commit